app/assets/stylesheets/components/_share-buttons.scss

```diff
@@ -0,0 +1,51 @@
+.share-buttons {
+  text-align: center;
+  padding-top: $base-spacing * 2;
+
+  h4 {
+    font-weight: $font-weight-normal;
+    color: $medium-gray-color;
+  }
+
+  a {
+    display: inline-block;
```

Comment:
Properties should be ordered border, border-radius, box-shadow, color, display, font-size, height, line-height, margin, opacity, outline, padding, width